src/components/CreatePollForm.vue

```diff
@@ -0,0 +1,304 @@
+        v-model.trim="pollData.description"
+        :class="{
+          'p-invalid': v$.pollData.description.$error,
+        }"
+        @input="v$.pollData.description.$touch"
+      />
+      <small v-if="v$.pollData.description.$error" id="poll_description" class="p-error">{{
+        v$.pollData.description.$errors[0].$message
+      }}</small>
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="poll_polledHouses">Список будинків: </label>
+      <div class="checkbox-section">
+        <div v-for="house of houses" :key="house.id" class="p-field-checkbox">
+          <Checkbox name="category" :id="house.id" :value="house" v-model="pollData.polledHouses" />
+          <label class="house-label" :for="house.id">
+            {{ house.address?.houseNumber }}, {{ house.address?.houseBlock }}, {{ house.address?.district }},
+            {{ house.address?.street }}
+          </label>
+        </div>
+      </div>
+      <small v-if="v$.pollData.polledHouses.$error" id="poll_polledHouses" class="p-error">{{
+        v$.pollData.polledHouses.$errors[0].$message
+      }}</small>
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="caledar-begin">Дата початку:</label>
+      <Calendar
+        id="caledar-begin"
+        dateFormat="dd.mm.yy"
+        v-model="startDate"
+        :showIcon="true"
+        :minDate="minDate"
+        @date-select="ChangeDate"
+      />
+    </div>
+    <div class="input-section">
+      <label class="dialog-item" for="calendar-finish">Дата закінчення:</label>
+      <InputText id="calendar-finish" disabled="true" :value="pollData.completionDate"></InputText>
+      <small id="apartment_area_help" class="p-warning">виставляється автоматично 15 днів з дати початку</small>
+    </div>
+    <div class="button-div">
+      <Button
+        id="save-button"
+        label="Додати опитування"
+        icon="pi pi-check"
+        autofocus
+        class="p-button-info"
+        type="submit"
+        value="Submit"
+      />
+      <Button
+        id="cancel-button"
+        label="Відмінити"
+        icon="pi pi-times"
+        class="p-button-outlined p-button-info"
+        @click="cancelEditing"
+      />
+    </div>
+  </form>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+import useVuelidate from '@vuelidate/core';
+import {
+  requiredValidator,
+  pollTitleLenghtValidator,
+  pollDescriptionLenghtValidator,
+  cyrillicLangTitleValidator,
+} from '@/utils/validators';
+import Button from 'primevue/button';
+import Textarea from 'primevue/textarea';
+import Checkbox from 'primevue/checkbox';
+import Calendar from 'primevue/calendar';
+import { StoreModuleEnum } from '@/store/types';
+import { HousesActionsEnum, HousesGettersEnum } from '@/store/houses/types';
+import { HouseModel } from '@/shared/models/house.model';
+import { PollsActionEnum } from '@/store/polls/types';
+import InputText from 'primevue/inputtext';
```

Comment:
wrong import order can lead to quality gate failure

please move this line above @ imports